mne/viz/tests/test_3d.py

```diff
@@ -576,7 +593,8 @@ def test_snapshot_brain_montage(renderer):
     """Test snapshot brain montage."""
     info = read_info(evoked_fname)
     fig = plot_alignment(
-        info, trans=None, subject='sample', subjects_dir=subjects_dir)
+        info, trans=Transform('head', 'mri'), subject='sample',
```

Comment:
we are deprecating the trans=None parameter? if so I don't think the what's new reflects this

Reply:
This was what @larsoner wanted, will update the `whatsnew`

Reply:
It just warns as well so I'm not sure if that counts as a depreciation, I think it should have warned before, an identity head->mri transform is rather odd to assume based on our conversation here https://github.com/mne-tools/mne-misc-data/pull/14

Reply:
Probably better to ensure it actually warns by using `trans=None` and wrapping in a
```
with pytest.warns(..., match='...'):
```
But as above, it should (in all cases/tests) only emit a warning if a head<->MRI transform is actually needed. Here it is, so the `pytest.warns` is useful to make sure it does the right thing.